datasets/castor_dataset.py

```diff
@@ -19,7 +20,7 @@ class CastorPairDataset(Dataset, metaclass=ABCMeta):
     LABEL_FIELD = None
 
     @abstractmethod
-    def __init__(self, path):
+    def __init__(self, path, additional_fields=None, examples_extra=None, load_ext_feats = False):
```

Comment:
Please don't use any spaces in default values for keyword args per PEP 8 specifications.

`load_ext_feats = False` => `load_ext_feats=False`